core/domain/classifier/lda_string_classfier.py

```diff
@@ -0,0 +1,598 @@
+# coding: utf-8
```

Comment:
OK, sounds good. Note, btw, that the name of the file is misspelled: classfier --> classifier.